src/templates/options.js

```diff
@@ -22,6 +22,6 @@ for (const [key, value] of Object.entries(options)) {
 
 /* <% if (options.langDir) { %> */
 export const ASYNC_LOCALES = {
-  <%= options.locales.map(l => `'${l.code}': () => import('../${relativeToBuild(options.langDir, l.file || l.iso)}' /* webpackChunkName: "lang-${l.code}" */)`).join(',\n  ') %>
+  <%= options.locales.filter(l => l.file).map(l => `'${l.code}': () => import('../${relativeToBuild(options.langDir, l.file)}' /* webpackChunkName: "lang-${l.file}" */)`).join(',\n  ') %>
```

Comment:
It's better to fail at build-time than at runtime when `l.file` is missing. I'm not sure if it would fail without the `filter` though...

The documentation states that [`file` is required](https://i18n.nuxtjs.org/options-reference#locales) when using lazy loading.

Reply:
Indeed we should fail earlier in module when lazy option is on (which i think is not current behavior of module)

Reply:
I've reverted for this PR